samples/dash-if-reference-player/app/sources.json

```diff
@@ -424,6 +424,30 @@
                     "moreInfo": "https://rdmedia.bbc.co.uk/testcard/simulcast/",
                     "provider": "bbc"
                 },
+                {
```

Comment:
One more thing I forgot: Can you please add these four streams to `test/functional-karma/config/subtitle`. Then they are included in the Karma functional tests that I execute when checking release candidates. For the config please use the same template as the other streams in `subtitle.js` e.g.

````
    {
        'name': 'Live Testcard - WOFF Font Download signalled with supplemental property descriptor',
        'url': 'https://rdmedia.bbc.co.uk/testcard/simulcast/manifests/avc-ctv-stereo-en-sfdt-woff.mpd',
        'moreInfo': 'any link,
        'provider': 'bbc',
        type: 'live',
        testcases: [Constants.TESTCASES.SIMPLE.SWITCH_TEXT],
    },
````

Reply:
I've added the four test streams to this now.